### PR TITLE
Added entry for SparrowGL

### PR DESCRIPTION
**Pull request by lprivette**
Hello, I left the `internalWebPropertyId` empty. Would you be able to generate one for this entry? I'm also happy to do it if you have any instructions. I feel bad having to ask you to do it everytime ha.

This entry is for the new Sparrow Great Lakes application: https://sparrow.wim.usgs.gov/great-lakes/

### CODE REVIEW

File: data/gaTable.yaml [hunk 1]
```diff
@@ -794,6 +794,27 @@
   analyticsContact: 'Hans Vraga'
   analyticsContactEmail: 'hvraga@usgs.gov'
 
+- longName: "Sparrow Great Lakes"
+  shortName: "SparrowGL"
+  accountName: WIM USGS
+  viewID: '166992080 '
```
Comment (wdwatkins) [inline data/gaTable.yaml]: can you remove the space in the string?  that might cause problems

**Comment by wdwatkins** [comment]
Internal web id is 166625472.  We're not using it for anything right now though, so not a big deal.

**Comment by lprivette** [comment]
Thanks! Sorry about the space. I've just committed that fix and the internal id.

**Comment by ldecicco-USGS** [comment]
http://internal.cida.usgs.gov/analytics/SparrowGL.html

Is this a new app (or newly added google analytic token)?

**Comment by wdwatkins** [comment]
yes

**Comment by ldecicco-USGS** [comment]
Than I think it's all good

**Pull request merged**